docs/choreolib/auto-routines.md

```diff
@@ -235,23 +243,23 @@ public AutoRoutine fivePieceAutoCompositionSeg(AutoFactory factory) {
               ampToC1.cmd(), intake(), aimFor(ampToC1.getFinalPose().orElseGet(Pose2d::new))),
           shootIfGp(),
           deadline(c1ToM1, waitSeconds(0.35).andThen(intake())),
-          new ConditionalCommand(
+          either(
               deadline(m1ToS1, aim()).andThen(shootIfGp()),
               deadline(m1ToM2, intake()).andThen(deadline(m2ToS1, aim()), shootIfGp()),
               yeGp() // if you aren't using the triggers API these wouldn't need a custom routine
-              ),
+          ),
           deadline(s1ToC2, intake(), aim()),
           shootIfGp(),
           deadline(c2ToC3, intake(), spinnup()),
-          shootIfGp())
-      .withName("fivePieceAuto");
+          shootIfGp()
+  ).withName("fivePieceAuto");
 
   return factory.commandAsAutoRoutine(ret);
 }
 ```
 
-## Creating an auto routine with composition and a monolithic trajectory
+### Creating an auto routine with composition and a monolithic trajectory
 
 ```java
-// Don't do this
+// This is not recommended for complex autos
```

Comment:
If we aren't going to provide an example maybe link to pathplanner saying they support this better?

Reply:
Theres no reason to link out, pathplanner does not support this any better than choreolib

Reply:
I know they did for 2024 has 2025 changed?

Reply:
No they didnt in 2024, event markers are an analog to triggers here so pathplanner does the monolithic trajectory with triggers. There is no better support for this style i can guarantee you.

Reply:
In 2024 with the pathplanner gui you could import the trajectory and use Named Commands to utilise this pattern. Is that still possible for 2025?

Reply:
What you're describing is triggers and a monolithic trajectory (unless you meant using a single named command at the start, with a long command group. But that was ill-advised even in 2024.)